Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.cpp

```diff
@@ -193,6 +193,24 @@ namespace Terrain
         maxHeightBounds = heightfieldAabb.GetZExtent() / 2.0f;
     }
 
+    float TerrainPhysicsColliderComponent::GetHeightfieldMinHeight() const
+    {
+        AZ::Aabb heightfieldAabb = GetHeightfieldAabb();
+
+        // Because our terrain heights are relative to the center of the bounding box, the min and max allowable heights are also
+        // relative to the center.  They are also clamped to the size of the bounding box.
+        return -(heightfieldAabb.GetZExtent() / 2.0f);
+    }
+
+    float TerrainPhysicsColliderComponent::GetHeightfieldMaxHeight() const
+    {
+        AZ::Aabb heightfieldAabb = GetHeightfieldAabb();
+
+        // Because our terrain heights are relative to the center of the bounding box, the min and max allowable heights are also
+        // relative to the center.  They are also clamped to the size of the bounding box.
+        return heightfieldAabb.GetZExtent() / 2.0f;
+    }
+
```

Comment:
Suggestion:  I know this wouldn't exactly simplify anything, but it might be nice to implement these two methods in terms of GetHeightfieldHeightBounds().  i.e.
```
{
    float minHeightBounds, maxHeightBounds;
    GetHeightfieldHeightBounds(minHeightBounds, maxHeightBounds);
    return minHeightBounds;
}
```
It's more lines of code, but it prevents the calculation logic itself from getting cut-and-pasted a few times, which is the part that has a higher likelihood of needing maintenance over time.

Reply:
Done.